src/Categories/Diagram/KernelPair.agda

```diff
@@ -0,0 +1,21 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Categories.Category.Core
+
+-- Kernel Pair - a Pullback of a morphism along itself
+-- https://ncatlab.org/nlab/show/reflexive+coequalizer
```

Comment:
is this link accurate? https://ncatlab.org/nlab/show/kernel+pair

Reply:
Oops! I must have pasted from the wrong window. Nice catch.